src/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+set(
+    sources
+    rdaddr.c
+    rdavl.c
+    rdcrc32.c
+    rdkafka.c
+    rdkafka_assignor.c
+    rdkafka_broker.c
+    rdkafka_buf.c
+    rdkafka_cgrp.c
+    rdkafka_conf.c
+    rdkafka_event.c
+    rdkafka_feature.c
+    rdkafka_metadata.c
+    rdkafka_metadata_cache.c
+    rdkafka_msg.c
+    rdkafka_offset.c
+    rdkafka_op.c
+    rdkafka_partition.c
+    rdkafka_pattern.c
+    rdkafka_queue.c
+    rdkafka_range_assignor.c
+    rdkafka_request.c
+    rdkafka_roundrobin_assignor.c
+    rdkafka_subscription.c
+    rdkafka_timer.c
+    rdkafka_topic.c
+    rdkafka_transport.c
+    rdlist.c
+    rdlog.c
+    rdports.c
+    rdrand.c
+    rdregex.c
+    rdstring.c
+    snappy.c
+    tinycthread.c
+)
+
```

Comment:
sasl seems to be missing.
Also do a master rebase, the sasl files changed names.